core-api/src/main/java/org/silverpeas/core/util/DateUtil.java

```diff
@@ -68,6 +69,9 @@
   public static final FastDateFormat ICALDATE_FORMATTER;
   public static final FastDateFormat ICALUTCDATE_FORMATTER;
   public static final FastDateFormat ISO8601_FORMATTER;
+  private static final DateTimeFormatter CUSTOM_FORMATTER =
+      DateTimeFormatter.ofPattern("yyyy/MM/dd");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "yyyy/MM/dd" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)